database/migrations/2019_08_07_070216_add_last_post_at_to_beatmap_discussions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddLastPostAtToBeatmapDiscussions extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('beatmap_discussions', function (Blueprint $table) {
+            $table->timestamp('last_post_at')->nullable();
+            $table->index(['user_id', 'updated_at']);
```

Comment:
Upon closer inspection this probably doesn't belong here (wrong migration name etc). Also missing drop index.